mozartdata/transforms/fact/aftership_rmas.sql

```diff
@@ -0,0 +1,183 @@
+        else
+            'other'
+    end                                                            as rma_status
+  , rmas.approved_at::date                                         as approved_date
+  , rmas.expired_at::date                                          as expired_date
+  , rmas.rejected_at::date                                         as rejected_date
+  , rmas.refunded_at::date                                         as refund_date
+  , rmas.resolved_at::date                                         as resolved_date
+  , case
+        when
+            aftership_org like '%warranty'
+            then
+            'warranty'
+        when
+            aftership_org like '%returns%'
+            then
+            case
+                when
+                    checkout_upsell_total != 0
+                    then
+                    'upsell'
+                when
+                    exchange_total_incl_tax_amount is null
+                        and checkout_upsell_total = 0
+                    then
+                    'refund'
+                when
+                    exchange_total_incl_tax_amount is not null
+                        and est_refund_amount = 0
+                        and checkout_upsell_total = 0
+                    then
+                    'exchange'
+                when
+                    exchange_total_incl_tax_amount is not null
+                        and est_refund_amount != 0
+                        and checkout_upsell_total = 0
+                    then
+                    'downsell'
+                else
+                    'unknown rma type'
+            end
+        else
+            'unknown organization'
+    end                                                            as rma_type
+  , rmas.return_total_with_tax_currency                            as currency
+  , rmas.return_total_with_tax_amount -
+    zeroifnull(rmas.return_tax_amount)                             as amount_product_total
+  , zeroifnull(rmas.return_tax_amount)                             as amount_tax_total
+  , rmas.refund_destination                                        as refund_payment_destination
+  , rmas.est_refund_amount - (zeroifnull(rmas.return_tax_amount) -
+                              zeroifnull(rmas.exchange_tax_total)) as amount_product_refunded
+  , zeroifnull(rmas.return_tax_amount) -
+    zeroifnull(rmas.exchange_tax_total)                            as amount_tax_refunded
+  , zeroifnull(rmas.exchange_total_incl_tax_amount) -
+    zeroifnull(rmas.exchange_tax_total)                            as amount_product_exchanged
+  , zeroifnull(rmas.exchange_tax_total)                            as amount_tax_exchanged
+  , rmas.exchange_order_number                                     as exchange_order_id_edw
+  , rmas.checkout_upsell_currency                                  as upsell_currency
+  , rmas.checkout_upsell_total                                     as amount_total_upsell
```

Comment:
apologies. I told you the wrong name for this column
```suggestion
  , rmas.checkout_upsell_total                                     as amount_upsell_total
```